core/controllers/admin.py

```diff
@@ -16,6 +16,7 @@
 
 import logging
 
+import random
```

Comment:
Move this below "import logging" so that all stdlib imports are in one place. Have a blank line below this to separate it from jinja2 which is not a stdlib import.